lib/matplotlib/tests/test_pickle.py

```diff
@@ -1,6 +1,5 @@
 from io import BytesIO
 import pickle
-import platform
```

Comment:
This is just to make flake8 happy - not part of this PR, but flake8 now seems to check all files, not just modified.